cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -75,6 +76,8 @@
     protected static final String ENV_VAR_KAFKA_CONNECT_TLS_AUTH_KEY = "KAFKA_CONNECT_TLS_AUTH_KEY";
     protected static final String ENV_VAR_KAFKA_CONNECT_SASL_PASSWORD_FILE = "KAFKA_CONNECT_SASL_PASSWORD_FILE";
     protected static final String ENV_VAR_KAFKA_CONNECT_SASL_USERNAME = "KAFKA_CONNECT_SASL_USERNAME";
+    protected static final String ENV_VAR_KAFKA_CONNECT_SASL_PASSWORD = "KAFKA_CONNECT_SASL_PASSWORD";
```

Comment:
This variable seems to be unused. Do we need it for anything?

Reply:
Forgot to remove this variable .. We tested in password string before make the secret files, I cleaned up password string related but missed this place.
Will fix this.

Reply:
code updated, please review ..